### PR TITLE
Adds "dirty" marker to window title. Closes #625

### LINKED ISSUE #625: Add dirty marker also to window title
I believe it would be useful to have DIRTY_MARKER (*) added also to window title whenever project gets dirty as it would improve "visibility" of the dirty state. Moreover, contrary to in-app project tree, application window title is usually also visible in system taskbar, so one can see the project state without opening the app window.

### PR DESCRIPTION
**Pull request by MarcinOrlowski**

### CODE REVIEW

**Review by BFH-ktt1 — CHANGES_REQUESTED** [review]
I think it is a typo, but I think you mean 
dirtyMarker and not dirtyNarker, just a detail

**Comment by MarcinOrlowski** [comment]
Indeed. Fixed. Thanks

**Review by MarcinOrlowski — COMMENTED** [review]
Typo fixed.

File: CHANGES.md [hunk 1]
```diff
@@ -15,6 +15,7 @@
   * Replace DarkLaf with FlatLaf for better compatibility.
   * Adds "Rotate Left" context menu action.
   * Display "Too few inputs for table" if Karnaugh Map has only 1 input.
+  * Project "Dirty" state is now also refrected in window title with "*" marker.
```
Comment (maehne) [inline CHANGES.md:18]: Typo: "refrected" -> "reflected"

**Pull request merged**